runtime/lua/vim/lsp/util.lua

```diff
@@ -799,12 +798,116 @@ function M.convert_input_to_markdown_lines(input, contents)
   return contents
 end
 
+local function parameter_range_in_signature(signature, active_parameter)
+  local from
+  local to = 0
+  for i = 0,#signature.parameters-1,1 do
```

Comment:
Same comment, spaces and the iterate interval is unnecessary

Reply:
fixed